src/hook/useTodo.js

```diff
@@ -0,0 +1,64 @@
+import { useCallback, useState } from "react";
+import { createTodoApi, deleteTodoApi, getTodosApi, updateTodoApi } from "../apis/api";
+import useToast from "./useToast";
+
+const useTodo = () => {
+  const toast = useToast();
+  const [todoList, setTodoList] = useState([]);
+
+  const fetchTodo = async () => {
+    try {
+      const data = await getTodosApi();
+      setTodoList(data);
+    } catch (error) {
+      console.error("패치 오류");
+    }
```

Comment:
여기서 useHttpError 사용할 수 있겠네요 아래 catch들 모두 해당하는 내용입니다!
```suggestion
    } catch (error) {
      showErrorToast(error)
    }
```